spec/features/api_spec.rb

```diff
@@ -1,5 +1,21 @@
 require 'spec_helper'
 
+shared_examples_for "api request" do
+	context "CORS requests" do
+		it "should set the Access-Control-Allow-Origin header to allow CORS from anywhere" do
```

Comment:
The recommended best practice for naming `it` blocks is to drop the word `should` and use the present tense for the following verb, as in `it 'sets the Access-Control....'`